### DISCUSSION TITLE
Schematic Placeholder

### DISCUSSION THREAD

**Discussion by andr3eee1** [BG-Software-LLC/SuperiorSkyblock2#1829] [Pending Feature Requests] [answered]
I would like a placeholder that would return to you what type of schematic a player's island is.

So that I can check if somehow that placeholder is a specific type, for example, oneblock, let's say I make a schematic for oneblock, which is used by the oneblock Addon. I can check if the island I have is of that schematic or not.

**Comment by Lemvs** [comment, marked as answer]
@OmerBenGera added
